cli/src/main/java/com/expediagroup/streamplatform/streamregistry/cli/command/delete/EntityClient.java

```diff
@@ -191,9 +192,11 @@
 
   static class Factory {
     @Option(names = "--streamRegistryUrl", required = true) String streamRegistryUrl;
+    @Option(names = "--username", required = true) String username;
```

Comment:
For OSS users, both here and in `GraphQLEventSenderAction` `username` and `password` should be optional.

They'd also be better named as `streamRegistryUsername` and `streamRegistryPassword` as they're used as a `@Mixin` with other options in the CLI so it should be clear what they are the username and password for.